lib/get-category-path.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var path = require('path'),
+  utility = require('./utility');
+
+
+/**
+ * Determines the category directory structure from the pattern's data
+ *
+ * @param {Object} patternObject  individual pattern's data (from /pattern-name/pattern.yml)
+ * @param {Object} options  pattern-importer options
+ *
+ * @return {String} categoryPath  internal path url to category
+ *
+ * @requires ./lib/utility
+ * @requires ./lib/category-name-converter
+ */
+exports.getCategoryPath = function (patternObject, options) {
```

Comment:
@e2tha-e even though it's temporary, I've added an eslint to this repo. It comes from drupal:
https://www.drupal.org/node/1955232
and includes these lines:

```

    "amd": true, // added to compensate for node
    "node": true // added to compensate for node
```

for node. I'll do some cleanup where needed. Shouldn't take long.